tool/release/notes/NotesCreate.kt

```diff
@@ -29,28 +29,32 @@ import kotlin.text.Regex.Companion.escapeReplacement
 fun main(args: Array<String>) {
     val bazelWorkspaceDir = Paths.get(getEnv("BUILD_WORKSPACE_DIRECTORY"))
     val githubToken = getEnv("NOTES_CREATE_TOKEN")
-    if (args.size != 5) throw RuntimeException("org, repo, commit, version, and template must be supplied")
-    val (org, repo, commit, version, templateFileLocation) = args
+    if (args.size != 6) throw RuntimeException("org, repo, commit, version, template, and output destination must be supplied")
+
+    operator fun <T> Array<T>.component6() = this[5]
```

Comment:
`Array<T>` only has up to `.component5()` by default. This is needed to enable destructuring on the next line.